default.nix

```diff
@@ -1,41 +0,0 @@
-{ mkDerivation, aeson, aeson-pretty, async, base, bytestring
```

Comment:
Can we keep the `default.nix` file, but change it to just import `release.nix`, or symlink it? That way a simple `nix-build` with no arguments will build the project.

Reply:
OK; 7e9c5563e7aaa7360078d99da766a95ee5401824 makes `default.nix` a symlink to `release.nix`.